src/DaPigGuy/PiggyCustomEnchants/enchants/armor/TankEnchant.php

```diff
@@ -23,14 +23,6 @@ class TankEnchant extends ReactiveEnchantment
     /** @var string */
     public $name = "Tank";
 
-    /**
-     * @return array
-     */
-    public function getReagent(): array
-    {
-        return [EntityDamageByEntityEvent::class];
-    }
-
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 21, so I commented this line